src/helpers/wallet.helper.js

```diff
@@ -62,6 +64,7 @@ export function getWalletConfigFromSeed({
   multisigData = null,
   passphrase = null,
   allowPassphrase = false,
+  scanPolicy = null,
```

Comment:
[❓ question] Shouldn't the `scanPolicy` be validated here?

Reply:
We validate on the route.
Any unsupported policy returns `{"success":false,"message":"Address scanning policy bla is not supported."}`
When using gap-limit we validate the `gap-limit` parameter and when using index-limit we validate the start and end limit parameters.

Reply:
Great! Thank you!